demo/src/initLoaders.js

```diff
@@ -1,7 +1,7 @@
 import { createTiffPyramid, createZarrPyramid } from '../../src';
 
 export async function initPyramidLoader(type, { channelNames, url, minZoom }) {
```

Comment:
Again, not sure if we should just make another function.  Probably doesn't matter here since this doesn't get released but the point is still important to discuss.

Reply:
I guess my question was more around semantics/API - we have this `minZoom` parameter and the word “pyramid” floating around a lot even though these functions now serve static images which don’t need them.  This is my concern with a lot of this stuff.

Reply:
I tried to address it but it might just require an audit outside this PR